test/genbank/test_region.py

```diff
@@ -187,3 +187,71 @@ def test_save(self):
         actual_row_count = len(cursor_result.fetchall())
 
         self.assertEqual(expected_row_count, actual_row_count)
+
+    def test_set_record_category(self):
+        """Tests whether region category is set correctly"""
+        core = ProtoCore(None, 1, 0, 100, False, "T1PKS", "PKS")
```

Comment:
to avoid any confusion in the future, perhaps add a comment line with smth like 'assume fully parsed protocore, as protocores dont natively have categories". unsure if its really needed, but did got me confused for a second or 30 xD